packages/core/src/browser/xhrProxy.spec.ts

```diff
@@ -120,13 +124,36 @@ describe('xhr proxy', () => {
         expect(request.status).toBe(200)
         expect(request.startTime).toEqual(jasmine.any(Number))
         expect(request.duration).toEqual(jasmine.any(Number))
+        expect(request.isAborted).toBe(false)
         expect(xhr.status).toBe(0)
         expect(onReadyStateChange).toHaveBeenCalled()
         done()
       },
     })
   })
 
+  it('should track aborted requests', (done) => {
```

Comment:
it could be nice to test different abort scenarios, I think of:
- abort call before state DONE
- abort call after state DONE but before loadend
- abort call after loadend

Reply:
Added as e2e tests.